packages/web-components/@sfx/search-box/test/unit/search-box.test.ts

```diff
@@ -0,0 +1,239 @@
+import { SEARCHBOX_EVENT } from '../../src/utils';
+import { expect, spy, stub } from '../utils';
+import SearchBox from '../../src/search-box';
+import { TemplateResult, html, LitElement } from 'lit-element';
+
+describe('SearchBox Component', () => {
+  let searchbox;
+  let windowDispatchEvent;
+
+  beforeEach(() => {
+    searchbox = new SearchBox();
+    windowDispatchEvent = spy(window, 'dispatchEvent');
+  });
+
+  afterEach(() => {
+    windowDispatchEvent.restore();
+  });
+
+  /*
+   * To be replaced with Base
+   */
+  describe('Constructor', () => {
+    it('should extend LitElement', () => {
+      expect(searchbox).to.be.an.instanceof(LitElement);
+    });
+  });
+
+  describe('connectCallback', () => {
+    it('should add an autocomplete_hover eventListener to the window', () => {
+      const windowAddEventListener = spy(window, 'addEventListener');
```

Comment:
This should also probably be a stub because you don't want it to actually listen for events.